### ISSUE TITLE
wxWidgets play button state (Trac #2)

### ISSUE THREAD

**Issue by vBm** [vBm/trac#2]
wxWidgets interface "Play" button does not turn anymore to "Pause" when needed

Migrated from https://trac.videolan.org/ticket/2
```json
{
    "status": "closed", 
    "changetime": "2007-03-17T16:20:13", 
    "description": "wxWidgets interface \"Play\" button does not turn anymore to \"Pause\" when needed", 
    "reporter": "zorglub", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "2007-03-17 16:20:13+00:00", 
    "component": "wxWidgets interface", 
    "summary": "wxWidgets play button state", 
    "priority": "high", 
    "platform": "", 
    "version": "master git", 
    "time": "2005-05-05T18:59:38", 
    "keywords": "", 
    "milestone": "", 
    "owner": "dionoea", 
    "difficulty": "", 
    "type": "defect", 
    "wip": "", 
    "severity": "normal"
}
```

**Comment by vBm** [comment]
Trac update at 20050511T10:48:23: courmisch commented:

> It's never worked on Linux. On Windows, it used to work with wx 2.4

**Comment by vBm** [comment]
Trac update at 20050515T17:57:15: 
* dionoea changed status from "new" to "assigned"
* dionoea changed owner from "gbazin" to "dionoea"

**Comment by vBm** [comment]
Trac update at 20050515T18:04:34: 
* dionoea changed status from "assigned" to "closed"
* dionoea commented:

> (In [11024]) fix play/pause button state ... gruiiiikkk
> Closes #2


* dionoea changed resolution from "" to "fixed"
